sample-app-cicd/run.yaml

```diff
@@ -15,8 +14,8 @@ spec:
     - name: repo-url
       value: https://github.com/darthlukan/gitops-workshop.git
     - name: git-branch
-      value: tekton-dev
+      value: pipeline-workspaces
```

Comment:
To test against this PR, use this branch. Otherwise this can be pointed to master after the PR is merged.